src/ugeneui/src/project_support/ProjectImpl.cpp

```diff
@@ -229,6 +230,21 @@ void ProjectImpl::sl_onObjectRelationChanged(const QList<GObjectRelation> &previ
     }
 }
 
+void ProjectImpl::sl_objectLoadedStateChanged() {
+    Document *doc = qobject_cast<Document *>(sender());
+    CHECK(doc != nullptr, )
+    if (doc->isLoaded()) {
+        return;
+    }
+    foreach (GObject *obj, doc->getObjects()) {
+        UnloadedObject *uo = qobject_cast<UnloadedObject *>(obj);
```

Comment:
How it is possible that unloaded document has loaded object?
I think the check below is not needed.